### ISSUE TITLE
convert from PropertySet to Property

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/circuit-construction-kit-common#220]
PropertySet is deprecated and should be replaced by Property, see phetsims/axon#102. Since this repo hasn't been published yet, this should be done before RC testing and publishing.

**Comment by samreid** [comment]
I converted PropertySet => Property in the above commits.  I'll leave the `preventGetSet` on for a few more days in case we catch other unintended usages (in a separate issue).
